runelite-client/src/main/java/net/runelite/client/plugins/mta/enchantment/EnchantmentRoom.java

```diff
@@ -66,6 +66,7 @@ public void onGameStateChanged(GameStateChanged gameStateChanged)
 			if (!inside())
 			{
 				dragonstones.clear();
+				client.clearHintArrow();

```

Comment:
This would probably break all game content which uses the hint arrow

Reply:
Yeah, completely did not realize. I've changed it to clear the hint arrow whenever a dragonstone despawns and the dragonstone location was the current hint arrow location.